visualizations/store-map-viz/utils/dataFormatting.tsx

```diff
@@ -35,11 +35,17 @@ export const deriveStatus = (location) => {
 };
 
 export const formatValues = (location) => {
+  
+  const label_prefix=location.icon_label_prefix;
+  const label_suffix=location.icon_label_suffix;
+  const label_precision=location.icon_label_precision;
+
   Object.keys(location).forEach((key) => {
+    const isClusterData = key.includes("cluster_");
     const isTooltip = key.includes("tooltip_");
     const isIconLabel = key === "icon_label" && !key.includes("_precision");
 
-    if (isTooltip || isIconLabel) {
+    if (!isClusterData && (isTooltip || isIconLabel)) {
```

Comment:
If you can think of a better way to derive the cluster icon settings from the individual marker settings. i wanted to avoid adding any more config options and so chose to derive the settings from the data itself.